ml_tools/model/state.py

```diff
@@ -347,3 +379,35 @@ def series_to_pandas(state_series: List[StateSeries], features: List[str] = None
         series_dfs.append(df)
 
     return pd.concat(series_dfs)
+
+
+def pandas_to_series(df: pd.DataFrame) -> List[StateSeries]:
+    """ Convert a Pandas DataFrame into a List of StateSeries
+
+    "DataFrame index must be a MultiIndex with 'series_index' and 'state_index'"
```

Comment:
Why the quotes?

Reply:
Works for me.  I don't really care, just wondered if they did something special.